mmcv/ops/csrc/common/pytorch_mlu_helper.hpp

```diff
@@ -0,0 +1,26 @@
+/*************************************************************************
+ * Copyright (C) 2021 Cambricon.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+ * CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *************************************************************************/
+#ifndef PYTORCH_MLU_HELPER_HPP_
+#define PYTORCH_MLU_HELPER_HPP_
+
+#ifdef MMCV_WITH_MLU
+#include "aten.h"
+
+#define NFU_ALIGN_SIZE 128
+
+#define PAD_UP(x, y) (((x) / (y) + (int)((x) % (y) > 0)) * (y))
```

Comment:
1. This seems duplicates the PAD_UP in common_mlu_helper.hpp, can we just keep one of them?
2. Is '(x)' necessary? can we simply write it as `((x / y + (int)(x % y > 0)) * y)`? If not, we may write comments to explain what you want to prevent so that readers and future developers could easily understand your motivation

Reply:
1.  common_mlu_helper.hpp keeps the functions only for device, and it will include more device-only functions in the future. Therefore, we have to keep the compilation isolation between common_mlu_helper and pytorch_mlu_helper.hpp.
2. the parentheses can be eliminated.